cpp/test/Ice/objects/Test.ice

```diff
@@ -165,16 +165,16 @@ class Recursive
 
 class K
 {
-    Value value;
+    AnyClass value;
 }
 
 class L
 {
     string data;
 }
 
-sequence<Value> ValueSeq;
-dictionary<string, Value> ValueMap;
+sequence<AnyClass> classSeq;
```

Comment:
Should be `ClassSeq` or `AnyClassSeq`.

Reply:
I'll change it to `ClassSeq`, you're correct it should be capitalized.